src/styles/bloks/card.css

```diff
@@ -0,0 +1,125 @@
+/* main block parametrs */
+.card {
+  display: flex;
+  flex-direction: column;
+
+  max-width: 198px;
+  padding: 32px 0 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+/* card blocks elem with image parameters */
+.card__container-image {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+/* card blocks elem with info parameters */
+.card__info {
+  margin-bottom: 4px;
+  line-height: 18px;
+  padding: 0 16px;
+
+  font-weight: 500;
+  font-size: var(--font-size_m);
+  color: var(--font-color_main);
+}
+
+/* card blocks elem with producte code parameters */
+.card__product-code {
+  margin-bottom: 16px;
+
+  padding: 0 16px;
+
+  line-height: var(--line-height_s);
+  font-weight: 400;
+  font-size: var(--font-size_s);
+  color: var(--font-color_grey);
+}
+
+/* card blocks elem with rate parameters */
+.card__rate {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 16px;
+
+  margin-bottom: 24px;
+}
+
+/* card blocks elem with review parameters */
+.card__review {
+  line-height: var(--line-height_s);
+
+  font-weight: 400;
+  font-size: var(--font-size_s);
+  color: var(--font-color_main);
+}
+
+/* card blocks elems with price parameters */
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 16px;
+
+  margin-bottom: 16px;
+}
+
+.card__text-price {
+  font-weight: 400;
+  font-size: var(--font-size_m);
+  color: var(--font-color_grey);
+}
+
+.card__num-price {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--font-color_main);
+}
+
+/* card blocks elems with button parameters */
+.card__button {
+  margin: 0 16px;
+}
+
+.card__link-button {
+  display: block;
+  box-sizing: border-box;
+
+  width: 100%;
+  height: 38px;
+  line-height: 16px;
+
+  padding: 11px;
+
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+
+  background-color: var(--font-color_button);
+
+  border: 1px solid var(--font-color_button);
+  border-radius: 5px;
+}
+
+.card__link-button:hover {
+  color: var(--font-color_button);
+
+  transition: 0.3s;
```

Comment:
Will be better to specify properties, for which you'll use transition
By default it'll apply transition for all properties, that can affect in bad way on the website performance